app/models/work_type.rb

```diff
@@ -6,4 +6,12 @@
 class WorkType < ApplicationRecord
   validates_presence_of :name_feminine, :name_masculine
   validates_uniqueness_of :name_feminine, :name_masculine
+
+  enum schema_org_role: {
+    "author": "author",
+    "illustrator": "illustrator",
+    "translator": "translator",
+    "editor": "editor",
+    "contributor": "contributor",
+  }
```

Comment:
These were intentionally defined as strings and not as ints for easier inspection.